image/cli/mascli/functions/catalog_utils

```diff
@@ -105,6 +105,26 @@ function choose_mas_version() {
   esac
 }
 
+# Determine whether UDS or DRO Operator to use
+# -----------------------------------------------------------------------------
+# We automatically select the UDS or DRO operator to install in the
+# cluster based on the version of the catalog that is being used
+# TODO: Need to know the latest catalog version name
+function uds_dro_selection() {
+  case $MAS_CATALOG_VERSION in
+    # UDS/DRO selection
```

Comment:
This can be simplified significantly:

```
    # UDS/DRO selection
    # -------------------------------------------------------------------------
    v8-240111-amd64)
      BAS_PROVIDER="DRO"
      ;;
    *)
      BAS_PROVIDER="UDS" # All other catalogs use UDS
      ;;
  esac
```

Reply:
Agree, changed